query/compile.go

```diff
@@ -569,6 +569,9 @@ func (c *compiledField) compileExponentialMovingAverage(name string, args []infl
 	if len(args) >= 3 {
 		switch arg2 := args[2].(type) {
 		case *influxql.IntegerLiteral:
+			if name == "triple_exponential_average" && arg2.Val < 1 && arg2.Val != -1 {
+				return fmt.Errorf("%s hold period must be greater than or equal to 1", name)
```

Comment:
Make some mention here about -1 being the option to set it to the default value. Maybe zero should set it to the default value instead? I'm not really sure I care which one signals the default value as long as one of them does and the error message mentions it.